### PR TITLE
Feature/task 6 django app

### PR DESCRIPTION
**Pull request by jaagomezmu**

### CODE REVIEW

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]
Hey Javier, remember to point your MR to `develop` branch instead of `main`.

File: myblog/blog/tests.py [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+                                            )
+    def test_creation_blog_post_3(self, user_1):
+        post = BlogPost.objects.create(title = self.title,
+                                       body = 'This is the body of the third test',
+                                       author = user_1,
+                                       )
+        assert str(post) == self.title
+
+        
+
```
Comment (ejariza-evowill) [inline myblog/blog/tests.py:39]: You should use more descriptive names for testing methods, so it's easier for other team members to understand what these methods are testing.

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]

File: myblog/blog/tests.py (hunk 1 above)
Comment (jaagomezmu) [reply myblog/blog/tests.py:39]: Of course, I'll keep it in mind

Comment (ejariza-evowill) [reply myblog/blog/tests.py:39]: Could you make this change for this merge request, please? Remember that you can save your current work by committing to the branch you are working on and then switching to this branch to make the requested fix.

**Review by ejariza-evowill — CHANGES_REQUESTED** [review]

**Review by ejariza-evowill — APPROVED** [review]

**Pull request merged**